doc/constraints.md

```diff
@@ -0,0 +1,88 @@
+## Video Constraints
+
+Video constraints are bi-directional messages that are exchanged between a given
+endpoint and the bridge to control the exchanged video quality. Note that the
+intention is to augment the format in the future with frame-rate and potentially
+a max resolution/frame-rate.
+
+###### Message format to the bridge
+
+An endpoint can request different qualities for different endpoints, so it sends
+a map of endpoint ids -> video constraints back to the bridge. In JSON notation
+the message can be represented like this:
+
+    [{
+      'id': endpoint-1,
```

Comment:
Is this what the client sends? Where is the code?

In the bridge I see `ReceiverVideoConstraintsMessage` using a different format (without `preferred` properties)

Reply:
Thanks Boris, that's a good point, I'll make the JSON description consistent.